sympy/matrices/common.py

```diff
@@ -1208,6 +1208,51 @@ def entry(i, j):
             return kls.zero
         return kls._new(size, size, entry)
 
+    @classmethod
+    def Wilkinson(kls, n):
```

Comment:
```suggestion
    def wilkinson(kls, n):
```
I'd try to use lowercase for names